README.md

```diff
@@ -20,14 +20,28 @@ However, it is a good idea to consult name changes with other people - use pull
 ## Gradle
 Yarn uses Gradle to provide a number of utility tasks for working with the mappings.
 
+### `yarn`
+[`setupYarn`](#setupYarn) and download and launch the latest version of [Enigma](https://github.com/FabricMC/Enigma) automatically configured to use the merged jar and the mappings.
+
+Compared to launching Enigma externally, the gradle task adds a name guesser plugin that automatically map enums and a few constant field names.
```

Comment:
> name guesser plugin

could explain it a bit better, is this info needed?

Reply:
Now i just use the latter description. Should suffice.